sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -0,0 +1,366 @@
+<#
+  .SYNOPSIS
+    Performs the tasks needed to setup an Azure subscription for use with the Event Hubs client

```

Comment:
This will be referenced from the contributing guide, it is intended to provide resource creation for running of the Live tests.

I wanted to include it to leverage the chance to have some eyes on it.

Reply:
Could you add a copyright header as well?

Reply:
Good catch.   Added.